crates/subspace-primitives/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "subspace-primitives"
+description = "Runtime primitives for Subspace Network"
+license = "GPL-3.0-or-later"
+version = "0.1.0"
+authors = ["Subspace Labs <https://subspace.network>"]
```

Comment:
I see this not the first time, but it is, strictly speaking, incorrect. Subspace Labs and subspace.network are not the same things, the website is not the official website for Subspace Labs, it is for the network itself.

Reply:
Yeah, actually I copied that from `subspace-runtime`, perhaps we need to define a global rule for it and fix them all.